scrum-board-system/src/components/userstories/AddUserStoryModal.js

```diff
@@ -0,0 +1,216 @@
+import React, {useState, useEffect} from 'react'
+import { Button, Modal, FormControl, InputGroup, Dropdown, DropdownButton } from 'react-bootstrap'
+import { useForm,useFieldArray } from 'react-hook-form'
+import { connect } from 'react-redux'
+import { addUserStory } from '../../actions/userstories'
+import {fetchLabels} from '../../actions/labels'
+import ReactTags from 'react-tag-autocomplete'
+
+
+const Sprint = ({sprint, sprints, register }) => {
+  const [sprintValue, setSprintValue] = useState(sprint?.name)
+  const [sprintId, setSprintId] = useState(sprint?.id)
+
+  const onChange = (change) => {
+                               setSprintId(change)
```

Comment:
Not sure about formating, looks ugly hahaha, maybe linter could help us to fix this...